CHANGELOG.md

```diff
@@ -4,6 +4,52 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.13.0 (2023-11-27)
+
+This release contains performance improvements for compressed hypertables
+and continuous aggregates and bug fixes since the 2.12.2 release.
+We recommend that you upgrade at the next available opportunity.
+
+Starting from TimescaleDB 2.13.0, no Amazon Machine Images (AMI) are published. 
+If you previously used AMI, please use another [installation method]
+(https://docs.timescale.com/self-hosted/latest/install/). 
+
+**Features**
+* #5575 Add chunk-wise sorted paths for compressed chunks
+* #5761 Simplify hypertable DDL API
+* #5890 Reduce WAL activity by freezing compressed tuples immediately
+* #6050 Vectorized aggregation execution for sum() 
+* #6062 Add metadata for chunk creation time
+* #6077 Make Continous Aggregates materialized only (non-realtime) by default
+* #6130 Add CI check for incorrect catalog updates
+* #6177 Change show_chunks/drop_chunks using chunk creation time
+* #6178 Show batches/tuples decompressed during DML operations in EXPLAIN output
+* #6185 Keep track of catalog version
+* #6227 Use creation time in retention/compression policy
+* #6307 Add SQL function cagg_validate_query
+
+**Bugfixes**
+* #6188 Add GUC for setting background worker log level
+* #6222 Allow enabling compression on hypertable with unique expression index
+* #6240 Check if worker registration succeeded
+* #6254 Fix exception detail passing in compression_policy_execute
+* #6264 Fix missing bms_del_member result assignment
+* #6275 Fix negative bitmapset member not allowed in compression
+* #6280 Potential data loss when compressing a table with a partial index that matches compression order.
+* #6289 Add support for startup chunk exclusion with aggs 
+* #6290 Repair relacl on upgrade
+* #6297 Fix segfault when creating a cagg using a NULL width in time bucket function
+* #6305 Make timescaledb_functions.makeaclitem strict
+* #6332 Fix typmod and collation for segmentby columns
+
+**Thanks**
+* @fetchezar for reporting an issue with compression policy error messages
+* @jflambert for reporting the issue
+* @pdipesh02 for contributing to the implementation of the generalized hypertable API
+* @pdipesh02 for contributing to the implementation of this feature
+* @pdipesh02 for contributing to the implementation of this feature
```

Comment:
I guess we need to remove the duplicate entries and place a comment about what PRs Dipesh helped

Reply:
I rewrote the `Thanks` section. So far, it was just the raw output of the changelog script.